backend/wargamebackend/wargamelogic/gamelogic/objects.py

```
@@ -1,5 +1,6 @@
 from dataclasses import dataclass
-from ..models import *
+from wargamelogic.models import *
```

Ruff (0.12.2)

`from wargamelogic.models import *` used; unable to detect undefined names

(F403)